### PR TITLE
iptables-legacy for Raspian Buster

### LINKED ISSUE #28: Raspbian Buster - iptables
As pointed out in the official docs, Raspbian Buster needs to change to iptables-legacy for k3s:
https://rancher.com/docs/k3s/latest/en/advanced/#enabling-legacy-iptables-on-raspbian-buster

Should we add tasks for this?

### PR DESCRIPTION
**Pull request by rockaut**
Fixes #28 - Changing to iptables-legacy and ip6tables-legacy only for Buster

### CODE REVIEW

**Comment by rockaut** [comment]
Sorry, I don't find why lint fails - it runs locally for me without msgs.

File: roles/raspbian/tasks/main.yml [hunk 1]
```diff
@@ -16,8 +16,32 @@
     - ansible_facts.architecture is search("arm")
   register: boot_cmdline
 
+- name: Flush iptables before changing iptables-legacy
+  iptables:
+    flush: true
+  when:
+    - ansible_facts.distribution_release is search("buster")
```
Comment (St0rmingBr4in) [inline roles/raspbian/tasks/main.yml:23]: Does this needs to run on Debian as well or just Raspbian?

Comment (rockaut) [reply roles/raspbian/tasks/main.yml:23]: Dokumentation tells only for raspbian. Is Debian also called Buster?

Comment (rockaut) [reply roles/raspbian/tasks/main.yml:23]: Thought this role would be loaded only on raspbian. Didn't realize you can load it anywhere.
Might be another catch. Will implement an additional when-rule

Comment (rockaut) [reply roles/raspbian/tasks/main.yml]: Changed when to `- ansible_facts.lsb.description is match("[Rr]aspbian.*[Bb]uster")`.

**Pull request merged**